docs/features/README.md

```
@@ -26,4 +26,4 @@ The Contribux platform provides:
 - **Authentication Setup?** See [authentication.md](authentication.md)
 - **API Usage?** Check [api-documentation.md](api-documentation.md)
 - **API Testing?** Review [api-testing-guide.md](api-testing-guide.md)
-- **Authentication Testing?** See [../testing/nextauth-v5-testing-guide.md](../testing/nextauth-v5-testing-guide.md)
\ No newline at end of file
+- **Authentication Testing?** See [../testing/nextauth-v4-testing-guide.md](../testing/nextauth-v4-testing-guide.md)
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)